apps/arweave/src/ar_pool_worker.erl

```diff
@@ -0,0 +1,90 @@
+-module(ar_pool_worker).
```

Comment:
What about renaming this `ar_pool_client`? And maybe `ar_pool` -> `ar_pool_server`?

elsewhere we use the `x` and `x_worker` to describe a worker that handles tasks for `x`

Reply:
sounds good!